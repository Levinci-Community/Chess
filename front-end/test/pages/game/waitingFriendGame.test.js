import React from "react";
import { render } from "@testing-library/react";
import WaitingFriendGamePage from "../../../src/pages/game/waitingFriendGame";
import { BrowserRouter as Router } from "react-router-dom";

jest.mock("react-i18next", () => ({
    useTranslation: () => ({
      t: (key) => key,
    }),
  }));
  
  test("renders WaitingFriendGamePage snapshot", () => {
    const { container } = render(
      <Router>
        <WaitingFriendGamePage />
      </Router>
    );
    expect(container).toMatchSnapshot();
  });