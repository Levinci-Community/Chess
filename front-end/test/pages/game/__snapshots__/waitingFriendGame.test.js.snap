// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders WaitingFriendGamePage snapshot 1`] = `
<div>
  <div
    class="chakra-container css-u5dkod"
  >
    <div
      class="css-d5gdhc"
    >
      <div
        class="chakra-card css-1cdm8x8"
      >
        <h2
          class="chakra-heading css-zx6l2c"
        >
          games.send_link_to_your_friend
        </h2>
        <div
          class="css-k008qs"
        >
          <input
            aria-readonly="true"
            class="chakra-input css-1etgquo"
            placeholder="games.game_url"
            readonly=""
            value="http://localhost/"
          />
          <button
            class="chakra-button css-4xx2wk"
            type="button"
          >
            games.copy
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
